velox/exec/tests/TableWriteTest.cpp

```diff
@@ -93,6 +94,32 @@ class TableWriteTest : public HiveConnectorTestBase {
     return getRecursiveFiles(directoryPath).size();
   }
 
+  // Returns a table insert plan node.
+  PlanNodePtr createInsertPlan(
```

Comment:
I wonder if it would be more readable / flexible to have a helper method that creates InsertTableHandle.